moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -160,7 +160,7 @@ static QString decideStatusText(const collision_detection::CollisionWorld::Objec
 {
```

Comment:
i think changes to Rviz should be a follow-up PR to keep the size of this one manageable. feel free to create it now, though

Reply:
Sure? It would make testing this PR less convenient, and I hope the Rviz changes will stay pretty limited.

Reply:
its not a hard rule, just a would be nice. your argument is fine

Reply:
#1122 and the ToDo in the code make a good case for rethinking the Rviz changes and how to visualize these frames. Red dots are obviously not ideal. I will keep this in for testing and remove it before the merge.